.npmrc

```diff
@@ -1 +1,12 @@
-# //registry.npmjs.org/:_authToken=${NODE_AUTH_TOKEN}
\ No newline at end of file
+registry=https://identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/ 
+                        
+always-auth=true
+
+; begin auth token
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:username=IdentityDivision
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:_password=[BASE64_ENCODED_PERSONAL_ACCESS_TOKEN]
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/registry/:email=npm requires email to be set but doesn't use the value
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:username=IdentityDivision
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:_password=[BASE64_ENCODED_PERSONAL_ACCESS_TOKEN]
+//identitydivision.pkgs.visualstudio.com/Engineering/_packaging/MSAL-JS/npm/:email=npm requires email to be set but doesn't use the value
+; end auth token
```

Comment:
Did you mean to include this? Think it can be discarded.

Reply:
removed